src/main/java/com/openlattice/shuttle/util/Parsers.java

```diff
@@ -0,0 +1,64 @@
+package com.openlattice.shuttle.util;
+
+import org.joda.time.LocalDate;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Parsers {
```

Comment:
It would be better to take the raw `Object` in all of these helpers instead of a String because it's one less thing the user needs to do. You can handle the `String` conversion here. As an example, this is a helper `getAsString` function I wrote:

```
    public static String getAsString( Object obj ) {
        if ( obj != null && obj.toString() != null ) {
            return obj.toString().trim();
        }
        return null;
    }
```

... and an example of how I use it:

```
    public static String getAsDate( Object obj ) {
        String dateStr = getAsString( obj );
        if ( dateStr != null ) {
            try {
                Date date = ( new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss" ) ).parse( dateStr );
                return ( new SimpleDateFormat( "yyyy-MM-dd" ) ).format( date );
            } catch ( Exception e ) {
                e.printStackTrace();
                return null;
            }
            return dateStr;
        }
        return null;
    }
```

... another example:

```
    public static UUID getAsUUID( Object obj ) {
        String id = getAsString( obj );
        if ( id != null ) {
            try {
                return UUID.fromString( id );
            } catch ( Exception e ) {
                e.printStackTrace();
                // TODO: what do we do if the id is not a valid UUID?
                return null;
            }
        }
        return null;
    }
```